index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport"
+    content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <div class="welcome">Welcome</div>
+      <div class="menu-button">
+        <img src="resources/menu-icon.svg" alt="menu">
+      </div>
+    </nav>
+  </header>
+  <main class="headline-section">
+
+    <h1 class="hello">Hey there. I'm Soufiane
+      <br>
+      <span>
+        I'm a software developer
+      </span>
+    </h1>
+    <p class="supporting-text">I can help you build a
+      product , feature or website
+      Look
+      through some of my work and experience! If you like
+      what you
+      see and have a project you need coded,
+      don’t hesitate to contact me.</p>
+    <div class="contact-me">
+      <h3>Let's connect</h3>
+      <ul>
+        <li>
+          <a href="https://twitter.com/sboursen_dev">
+            <img src="resources/twitter-icon.svg"
+              alt="go to twitter">
+          </a>
+        </li>
+        <li>
+          <a href="https://www.linkedin.com/in/sboursen/">
+            <img src="resources/Linkedin-icon.svg"
+              alt="go to linkedin"></a>
+
+        </li>
+        <li>
+          <a href="#">
+            <img src="resources/medium-icon.svg"
+              alt="go to medium">
+          </a>
+
+        </li>
+        <li>
+          <a href="https://github.com/Sboursen">
+            <img src="resources/github-icon.svg"
+              alt="go to github">
+          </a>
+
+        </li>
+        <li>
+          <a href="#"><img
+              src="resources/angellist-icon.svg"
+              alt="go to angellist"></a>
+
+        </li>
+      </ul>
+    </div>
+  </main>
+</body>
```

Comment:
- [ ] Kindly fix the background colour as it is not the same with the one provided in the figma.

### See below your design
![your_design](https://user-images.githubusercontent.com/58692590/148250816-e39f43ce-391e-481b-b55e-89d5ce1be9db.png)

### See below the provided figma design
![figma](https://user-images.githubusercontent.com/58692590/148250954-cbaae0d2-724c-4819-b5aa-05cd918b7df2.png)

Reply:
Hi @Whoistolu!
Thank you for the feedback.